js/index.js

```diff
@@ -40,3 +41,107 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let logo2 = document.getElementById("cta-img");
+logo2.setAttribute('src', siteContent["cta"]["img-src"])
+
+let logo3 = document.getElementById("middle-img");
+logo3.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+const header = document.querySelector('header')
+header.style.backgroundColor = "Gainsboro"
+header.style.borderBottomLeftRadius = "10px"
+header.style.borderBottomRightRadius = "10px"
+const nav = document.querySelectorAll('a')
```

Comment:
navItems is a NodeList created by querySelectorAll, so you can iterate through it via loops:

```
navItems.forEach((link, index) => {
     link.textContent = siteContent['nav'][`nav-item-${index+1}`];
     link.style.color = 'green';
});
```

OR

```
for (let i=0; i < links.length; i++) {
  navItems[i].textContent = siteContent['nav'][`nav-item-${i+1}`];
  navItems[i].style.color = 'green';
};
```